cmd/applications.go

```diff
@@ -93,12 +122,177 @@ var listMethodsCmd = &cobra.Command{
 	},
 }
 
+var addApplicationCmd = &cobra.Command{
+	Use:   "add",
+	Short: "Add an application",
+	Long: `Add an application to PAS.
+	
+	Example Usage:
+	$ cybr applications add -a AppID -l "\\"`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client, err := pasapi.GetConfig()
+		if err != nil {
+			log.Fatalf("Failed to read configuration file. %s", err)
+			return
+		}
+
+		newApplication := pasapi.AddApplicationRequest{
+			Application: pasapi.Application{
+				AppID:               AppID,
+				Location:            Location,
+				Description:         Description,
+				AccessPermittedFrom: AccessPermittedFrom,
+				AccessPermittedTo:   AccessPermittedTo,
+				ExpirationDate:      ExpirationDate,
+				Disabled:            Disabled,
+				BusinessOwnerFName:  BusinessOwnerFName,
+				BusinessOwnerLName:  BusinessOwnerLName,
+				BusinessOwnerEmail:  BusinessOwnerEmail,
+				BusinessOwnerPhone:  BusinessOwnerPhone,
+			},
+		}
+
+		err = client.AddApplication(newApplication)
+		if err != nil {
+			log.Fatalf("Failed to add application '%s'. %s", newApplication.Application.AppID, err)
+			return
+		}
+
+		fmt.Printf("Successfully created application '%s'\n", newApplication.Application.AppID)
+	},
+}
+
+var deleteApplicationCmd = &cobra.Command{
+	Use:   "delete",
+	Short: "Delete an application",
+	Long: `Delete an application to PAS.
+	
+	Example Usage:
+	$ cybr applications delete -a AppID`,
+	Run: func(cmd *cobra.Command, args []string) {
+		client, err := pasapi.GetConfig()
+		if err != nil {
+			log.Fatalf("Failed to read configuration file. %s", err)
+			return
+		}
+
+		err = client.DeleteApplication(AppID)
+		if err != nil {
+			log.Fatalf("Failed to delete application '%s'. %s", AppID, err)
+			return
+		}
+
+		fmt.Printf("Deleted application '%s'\n", AppID)
+	},
+}
+
+var addApplicationAuthenticationMethodCmd = &cobra.Command{
+	Use:   "authn-add",
```

Comment:
When doing that I was getting weird results and it doesn't work as intended. 

I don't know if we need to make an `authnMethodCmd` and then made an `authnAddMethodCmd` and then nest `authnAddMethodCmd` inside of `authnMethodCmd`. 

I like having it flat because then I can just do:
```
cybr applications --help
```

And get all of the actions I can do on an application.

What are your thoughts?

Reply:
We could change it to `add-authn` or `delete-authn` so the full command would work

```
cybr applications delete-authn -a appID -i authnID
```

Reply:
Yeah, that does make sense.  I'd rather people be able to obtain easy help then having to keep bumping into subcommands like you have to in the Conjur CLI.  Ok, this is good!